docs/data-sources/groups.md

```diff
@@ -0,0 +1,41 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_groups Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Groups are how users are authorized to access Resources. For more information, see Twingate's documentation https://docs.twingate.com/docs/groups.
+---
+
+# twingate_groups (Data Source)
+
+Groups are how users are authorized to access Resources. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/groups).
+
+## Example Usage
+
+```terraform
+data "twingate_groups" "foo" {
+  name = "<your group's name>"
+}
+```
+
```

Comment:
Let's add a clarification comment below this to give some context as to why this is a plural data source.

`Group names are not constrained to be unique within Twingate, so it is possible that this data source will return multiple list items.`

Reply:
updated docs

Reply:
Ah, I see that this is within the Terraform example itself, and now I understand the `#` comment notation. Originally I was thinking that we should change this to move it up into the general description, but I think this works. Will approve this and the other similar changes and then we can merge everything.